classes/Carrier.php

```diff
@@ -203,6 +203,19 @@ public static function resetStaticCache()
         Cache::clean('Carrier::*');
     }
 
+    /**
+     * @param float $total_weight
+     * @return float
+     */
+    public static function adjustTotalWeight(float $total_weight): float
```

Comment:
I suggest naming it something like `addPackingWeight`.

Reply:
Method name changed